Editing notes using the github.dev editor.md

```diff
@@ -0,0 +1,63 @@
+---
+aliases: 
+- 
+tags:
+- seedling
+publish: true
+---
+
+# Editing notes using the github.dev editor
+
+%% Add a description below this line. It doesn't need to be long: one or two sentences should be a good start. %%
+
+This guide will cover the basics of using the github.dev editor for the [[Obsidian Hub]]. This editor is great when you are making quick edits that don't need any Obsidian-specific functionality.
+
+In particular, if you are browsing the Hub and you see a note that you would like to improve, this guide talks you through how to make the edit, using the "Edit in GitHub" button at the bottom of the note.
+
+## Opening the github.dev editor
+
+To open the editor you'll usually have two options: 
+1. On the Hub's [publish site](https://publish.obsidian.md/hub/00+-+Start+here), each note has an "Edit in GitHub" button in its footer.
+2. When you are in the Hub's [GitHub repository](https://github.com/obsidian-community/obsidian-hub), pressing <kbd>.</kbd> will take you to the editor. 
+
+Note that you will need a GitHub account to make edits, and if you aren't logged in to it, launching the github.dev editor will prompt you for your GitHub login details.
+
+## Making changes to the notes
+
+1. To make changes, you can select files from the file explorer (1), and make edits to the note (2) as you would on any text editor or paste any changes made in Obsidian to existing notes. To add new files, you can also drag from your system's file explorer to the file explorer in the editor (1).
+	![[github.dev editor.png]]
+2. Once you are ready to submit your changes, switch to the version control tab. You'll see the names of the files you've modified under the "Changes" section:
+    ![[github.dev editor version control.png|300]]
+3. Add the files you want to submit by pressing the <kbd>+</kbd> sign (2), and write a message describing the changes you've made(1): 
```

Comment:
Like, I know that there is sometimes a reason for not adding everything to a commit when working with git in general. But for the occasional contributor not familiar with git, would there ever be a scenario where you do make changes that you do *not* end up adding to the commit?

I think it would be easier to simply phrase instruct people to simply add everything (or use such an add everything button, if there is one).

Reply:
In VSCode, there is a `+` button to add them all, I wouldn't go into too much detail which could be confusing.